services/traefik/20.2.0/traefik.yaml

```diff
@@ -11,21 +11,24 @@ spec:
         kind: HelmRepository
         name: helm.traefik.io-traefik
         namespace: kommander-flux
-      version: 10.30.1
+      version: 20.2.0
   interval: 15s
   install:
     crds: CreateReplace
     remediation:
       retries: 30
   upgrade:
     crds: CreateReplace
+    # Revert the "force" flag in a future bump of traefik.
+    # This is needed here for now because of https://github.com/traefik/traefik-helm-chart/issues/685 that was introduced in v17.0.1.
+    force: true
```

Comment:
I imagine recreating the traefik deployment would result in some downtime and based on the comments on that issue, would it recreate the ELB (if running in AWS) causing the ingress address to change?

Reply:
assuming you are referring to this comment https://github.com/traefik/traefik-helm-chart/issues/685#issuecomment-1290677274 , i have tested upgrades manually and nothing is recreated. if the `Service` is deleted and recreated then yeah we will get a new ELB. But the "breaking" change has to deal with only immutable fields of Deployment so we will not `Recreate` the `Service` resource so the ELB will not be recreated.

Reply:
Haha!! Traefik is quick to respond. 
https://github.com/traefik/traefik-helm-chart/releases/tag/v20.3.0 was released this morning which lets us set a label override manually. I ll bump to it. :rocket: